### ISSUE TITLE
UE 1-3-1 Buffs

### ISSUE THREAD

**Issue by Doruk-Coskun** [captaincobbs/ESG-Mod#383]
Traitors reach quest node updated.

![image](https://user-images.githubusercontent.com/25507419/111622516-57d3bc80-87fa-11eb-9e40-c4f139dcd03b.png)

Punisher Drives is buffed 10% -> 15% Damage on Ships

![image](https://user-images.githubusercontent.com/25507419/111622674-8487d400-87fa-11eb-8bb8-ab2174abaded.png)

**Comment by Doruk-Coskun** [comment]
![image](https://user-images.githubusercontent.com/25507419/111623576-9ae25f80-87fb-11eb-83ad-ec5138603106.png)

Turned approval bonus of "Journey to the Stars" into an Empire Improvement. Buffed per science improvement approval bonus 3 -> 4 

UI Elements displayed in the screen shot are updated and fixed

**Comment by Doruk-Coskun** [comment]
TODO: Rework influence buyout of techs

**Comment by NingaublesEyes** [comment]
I've been playing through the UE a bit. I don't think I'm at 1.3.1 but I've been playing the UE and I have 2 thoughts, 1 suggestion and 1 idea. The idea isn't fleshed out yet but if you shift from Imperial to Sherydn or Mezari - could the final skill skill point (or possibly the faction skill tree) be adjusted to better emphasis those factions. 

The suggestion is this. I appear to be playing with the +100% influence bonus and I've definitely enjoyed playing the UE more than I expected. But I think they could use a little smoothing out. I like that their faction's ability is to cut corners that's pretty cool. I am not sure on the exact numbers for this but how about a unique building on their home system. "Seat of Empire" or even, "Szinda's Monument" +1.5 influence per system level. It doesn't sound like much but with the +100% bonus it's a flat bonus that increases over time 3/6/9/12. This would just be on the home system. Obviously that might not be accurate depending on when that 100 kicks in. So it might even be more than that, but I feel like the home system should generate a bit more influence. Not just aesthetically as the 'seat of power'. Originally I was thinking it should be something that could scale into a more ridiculous number like ... +1% influence on the home system.

**Comment by NingaublesEyes** [comment]
Also I should note that I am playing with a customize version for the Imperials. 
+1 Influence. 
+2 Ind on Happy. 
Collection bonus : Patriotic. I have yet to have the 50% influence bonus kick in and I'm on turn 59 / 150. I'm at 45 population and I'm only generating 270 influence right now. But I haven't felt particularly weak. I'll make a brief update to this comment when the influence bonus kicks in.

**Comment by NingaublesEyes** [comment]
So I expect you already have plans for this faction as presently it's not utilizing all or most of the pop & trait points.

But I definitely enjoyed playing with the patriotic collection affinity. I added a tikamin instead of a yashuo but I'm not even sure I have the bonus after 70 or so turns. I don't make as much influence as I could with the other bonuses but I basically always have troops to fill my ships with. I have 5 unspent points - I was considering trying them again with core worlds or another 5 points in manpower on ships. 

I was pretty starved for dust I find myself doing a lot of margin buying on all fronts. It's fun to cut so many corners or have a surge of fleets. I VERY MUCH dig the drop in influence that comes from a spike in ship production though. While ... I recognize this may not really "buff them" it creates a really thematic experience. When I was playing the Nakaleem I took a liking to them but started trying to customize them almost immediately trying to get the 'feel' of an ancient empire returning to the galaxy and found a version I find satisfying. I'm pretty much of the same opinion of this version of the UE.

![20210319103749_1](https://user-images.githubusercontent.com/52173864/111821729-37f6d400-88a0-11eb-829f-6ad645c4ca76.jpg)

**Comment by Doruk-Coskun** [comment]
> I've been playing through the UE a bit. I don't think I'm at 1.3.1 but I've been playing the UE and I have 2 thoughts, 1 suggestion and 1 idea. The idea isn't fleshed out yet but if you shift from Imperial to Sherydn or Mezari - could the final skill skill point (or possibly the faction skill tree) be adjusted to better emphasis those factions.
> 
> The suggestion is this. I appear to be playing with the +100% influence bonus and I've definitely enjoyed playing the UE more than I expected. But I think they could use a little smoothing out. I like that their faction's ability is to cut corners that's pretty cool. I am not sure on the exact numbers for this but how about a unique building on their home system. "Seat of Empire" or even, "Szinda's Monument" +1.5 influence per system level. It doesn't sound like much but with the +100% bonus it's a flat bonus that increases over time 3/6/9/12. This would just be on the home system. Obviously that might not be accurate depending on when that 100 kicks in. So it might even be more than that, but I feel like the home system should generate a bit more influence. Not just aesthetically as the 'seat of power'. Originally I was thinking it should be something that could scale into a more ridiculous number like ... +1% influence produced from the home system.

I have plans for zelevas incarnate so that it works for every system. Changing skill trees of mezari/UE/sheredyn is a really interesting idea and I like it. But due to implementation and balancing complications its not possible

**Comment by Arpharnator** [comment]
if you plan on making the zelevas incarnate be on every system make sure it does not feedback into itself ( ie : zelevas incarnate makes dust/ind from influence, but you receive inf from either dust or ind, resulting in an infinite loop)

**Comment by Doruk-Coskun** [comment]
![image](https://user-images.githubusercontent.com/25507419/112526586-2d15d500-8db3-11eb-95de-68ddd64bc371.png)
Zelevas Incarnate Unique updated to give additional 0.3 Science per Influence and its effect is extended to every system when built. Currently Zelevas Incarnate effect on production, dust and science are working but UI element indicating source of the increase is not displayed and will be fixed.

**Comment by Arpharnator** [comment]
check with mezari pop collection bonus to not get infinite loops ples

**Comment by Doruk-Coskun** [comment]
![image](https://user-images.githubusercontent.com/25507419/112526934-8aaa2180-8db3-11eb-904c-fd0ca955e8ac.png)
Journey to the Stars system improvement updated due to the split of approval effect from the improvement

**Comment by Doruk-Coskun** [comment]
![image](https://user-images.githubusercontent.com/25507419/112527103-baf1c000-8db3-11eb-94a4-17a82919262a.png)
Daily Slogan effect updated

**Comment by Arpharnator** [comment]
jeez man I didn't thought yelling would make babies

**Comment by Doruk-Coskun** [comment]
Production Buyout Formula Base cost reduced from 100 -> 25. New formula is:
Influence Cost = ((Production Cost) ^ 1.1 + 25) 
Science Buyout Formula Base cost reduced from 150 -> 25. New formula is:
Influence Cost = ((Science Cost) ^ 1.01 + 25)

**Comment by Arpharnator** [comment]
does that mean that buying out industry is 4 times cheaper, and buying out science is 7 times cheaper?

**Comment by Doruk-Coskun** [comment]
No, it means buying out early techs and improvements are easier since for the early improvements most of the influence cost comes from the base cost rather than the cost scale

**Comment by wakystuf** [comment]
I suspect that, due to the first term, it would be much less than that, like 50% reduction for really cheap stuff and a very small % for later techs and wonders. This seems like it makes influence better than ever at getting going on the basic infrastructure, but not much different at larger items

**Comment by wakystuf** [comment]
Yeah, what he said :)

**Comment by Doruk-Coskun** [comment]
Most of the Quest requirements are reworked and rebalanced to fit the pacing of the game.

**Comment by NingaublesEyes** [comment]
Wow these are some impressive changes. I may jump back on in a few weeks or
so I'm taking an Amplitude break. All the best. Y'all deserve props from
their staff for keeping the game dynamic and breathing new life into it.

**Comment by Doruk-Coskun** [comment]
Imperials Population Bonus 3 changed to 0.2 Influence per Industry Spent. 
Choosing to stay Imperial positions at Quest 3 rewards 0.2 Influence per Industry Spent. With collection bonuses, a maximum of 0.5 Influence per Industry spent can be achieved. 
Switching gives +20% Troop Health and Damage rather than providing 25 Influence per CP killed. 
Emperors Module, a quest module awarded in Sheredyn path, effect is buffed to provide 100 projectile and energy defense to each ship in the flotilla in addition to +10% Morale Effect.

**Comment by wakystuf** [comment]
This appears to be Finished, but also Open. Should I go ahead and move it to Closed and put it on the wiki?

**Comment by captaincobbs** [comment]
It's waiting for me to get a chance to review it, I've been doing homework. Go ahead and put this on the wiki, we can change it if needed.

**Comment by wakystuf** [comment]
OK, this is now added to the wiki. I haven't played the UE in forever, so can you please clarify which questline this refers to (sheredyn or mezari)?
Switching gives +20% Troop Health and Damage rather than providing 25 Influence per CP killed.

**Comment by captaincobbs** [comment]
> @captaincobbs I accepted the merge request due to time limitations till tonight's game. Please make sure you check it when you can in any way.

It looks fine to me

**Comment by Doruk-Coskun** [comment]
> Switching gives +20% Troop Health and Damage rather than providing 25 Influence per CP killed.

@wakystuf That one is Sheredyn

**Comment by wakystuf** [comment]
Great. Documented.
